snmp/backupninja.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+import io
+import re
+import gzip
+import os
+import json
+
+LOGFILES = [
+    "/var/log/backupninja.log",
+    "/var/log/backupninja.log.1.gz",
+]
+
+# Last status
+last_actions = 0
+last_fatal = 0
+last_error = 0
+last_warning = 0
+
+def main():
+    get_backupninja_state()
+    output = {'actions': last_actions, 'fatal': last_fatal, 'error': last_error, 'warning': last_warning}
```

Comment:
your json isn't complete/valid for LibreNMS

structure:
version: (1 in your case, so later upgrades are possible)
error: (by default 0, but can have values on error)
error_string: (by default '', but can contain error message on error)
data: <your dictionary with data>